app/src/main/res/values/colors.xml

```diff
@@ -194,6 +194,9 @@
     <color name="calendarHeaderBackground">@color/polar_bear</color>
     <color name="manyAvatarsAdditionalPeopleBackground">@color/iconColorSecondaryText</color>
 
+    <!-- Snooze -->
+    <color name="snoozeColor">@color/coral_light</color>
```

Comment:
Either put this next to the scheduledColor in iconColor region, 
Or create a region for scheduled color down here